File: app/src/main/java/me/mitul/aij/aij/AijExplorerActivity.java
package me.mitul.aij.aij;

import android.os.Bundle;
import android.view.WindowManager;

import androidx.appcompat.app.AppCompatActivity;
import androidx.fragment.app.FragmentTransaction;

import me.mitul.aij.utils.PlaceHolderFragment;
import me.mitul.aij.R;

public class AijExplorerActivity extends AppCompatActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_explore_aij);
        getWindow().setFlags(WindowManager.LayoutParams.FLAG_FULLSCREEN, WindowManager.LayoutParams.FLAG_FULLSCREEN);
        FragmentTransaction transaction = getSupportFragmentManager().beginTransaction();
        transaction.replace(R.id.fragment_container, PlaceHolderFragment.newInstance(getIntent().getIntExtra(getString(R.string.item_id), 0)));
        transaction.commit();
    }
}

File: app/src/main/java/me/mitul/aij/aij/AijExplorerActivity.kt
package me.mitul.aij.aij

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import me.mitul.aij.R
import me.mitul.aij.aij.FragmentCommon.Companion.newInstance

class AijExplorerActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.aij_activity_explore)

        supportFragmentManager.beginTransaction().replace(
            R.id.fragment_container,
            newInstance(intent.getIntExtra("item_id", 0))
        ).commit()
    }
}
